Dockerfile.build

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.6
+
+ENV GO15VENDOREXPERIMENT=1
+ENV GOPATH=/go
+ENV PATH=$PATH:$GOPATH/bin
+
+COPY . /go/src/github.com/drone/drone
+WORKDIR /go/src/github.com/drone/drone
+
+RUN make deps
```

Comment:
Point of separating them out was to generate intermediate images so you could re-run the build more easily (I'll have to separate the `ADD` statements to make this work properly, actually).